src/simd.h

```diff
@@ -31,6 +31,8 @@
 #ifndef __NTTEC_SIMD_H__
 #define __NTTEC_SIMD_H__
 
+#ifdef NTTEC_USE_SIMD
```

Comment:
Or maybe we could use

#ifndef NTTEC_USE_SIMD
#error "cannot include simd.h if SIMD is not enabled".
#endif // #ifndef NTTEC_USE_SIMD

Reply:
NTTEC_USE_SIMD can be undefined. It's not an error.

Reply:
Depends how you see things, but IMHO including simd.h with simd disabled could be considered as an error, no?

Reply:
But we can keep it like that as well.

There are two approach:
- you always include simd.h, but simd.h is potentially empty (if SIMD is disabled)
- the content of simd.h is always the same, and the other files include it conditionnally

e.g: 

```
#ifdef NTTEC_USE_SIMD
#include "simd.h"
#endif NTTEC_USE_SIMD
```
In gf_nf4.cpp.

The "advantage" of the second approach is that simd.h content never change and anyway we already have a test on NTTEC_USE_SIMD in gf_nf4.cpp so the include could be grouped with the existing one.

Reply:
I thought also your second way. It's actually updated in this way. Thanks :)